sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -982,7 +1035,9 @@ export class MessageReceiver extends LinkEntity {
               await this.close();
             } else {
               if (this._receiver && this.receiverType === ReceiverType.streaming) {
-                this._receiver.addCredit(this.maxConcurrentCalls);
+                if (!this._stopReceivingMessages) {
+                  this.addCredit(this.maxConcurrentCalls);
+                }
```

Comment:
No need for the if-check?
`this.addCredit` already checks for that, right?

Reply:
Correct!

Reply:
Fixed.